projects/observability/src/shared/components/topology/d3/d3-topology.ts

```diff
@@ -167,7 +167,8 @@ export class D3Topology implements Topology {
       .append('svg')
       .classed(D3Topology.SVG_CLASS, true)
       .attr('width', '100%')
-      .attr('height', '100%');
+      .attr('height', '100%')
+      .style('padding-top', '24px');
```

Comment:
Is this padding we want everywhere? We use this both as a full screen and embedded widget in different places, so may make more sense to put it on the specific usage

Reply:
Yes, it applies everywhere.

Reply:
I think this will end up just looking weird - especially on topology when nodes are draggable, adding padding here means nodes will magically cut off. 
![image](https://user-images.githubusercontent.com/45047841/110708616-cc31ae80-81af-11eb-9212-0b76ecfbb469.png)

So I guess I'm not sure what we're trying to accomplish with the padding

Reply:
The approach was changed.